### ISSUE TITLE
[v4][iOS] topBar background color not working on iOS 12

### ISSUE THREAD

**Issue by selyon** [wix/react-native-navigation#5722]
### Issue Description

In my react native application i have a topBar with a custom background color and a custom Title component inside it.
With react-native-navigation 3.7.0 it works both on old versions of iOS (12 in my case) and on iOS 13.

I tried to update react-native-navigation to version 4.0.2 but now the topBar doesn't work anymore.
On iOS 13 nothing changed, it's working. But on iOS 12 the background color is not displayed.
I see my custom title component, with its background and all the other parts, but the rest of the topBar is white. It turns black if I set 
{
    topBar: {
        barStyle:  'black',
        ....
    }
}

Below is my topBar configuration object

### Steps to Reproduce / Code Snippets / Screenshots
Install v4 and use a configuration like this one

```js
topBar: {
        title: {
          component: {
            name: 'topbar',
          },
        },
        background: {
          color: '#00B5B8',
        },
        drawBehind: false,
        visible: true,
        animate: false,
} 
```

---
### Environment
* React Native Navigation version: 4.0.2
* React Native version: 0.61.5
* Platform(s) (iOS, Android, or both?): iOS 12
* Device info (Simulator/Device? OS version? Debug/Release?): Both on real device and emulator with iOS version 12 (and less I think but not tested). Both debug and release build

**Comment by drewhamlett** [comment]
I think it was fixed here.  https://github.com/wix/react-native-navigation/commit/7cf208353f22a4f07760f166ba0afd5af24f2e91

**Comment by guyca** [comment]
Hey @selyon This should have been fixed in v4.0.3

**Comment by selyon** [comment]
Yes with version 4.0.3 it's working good. The changelog should be updated.
Thank you.
